plus/async/sleep.js

```diff
@@ -4,4 +4,16 @@
  * @param {number} sec the seconds to wait
  * @returns {Promise} the promise which resolves when wait if over
  *                    or rejects if parameter is not correct
- */
\ No newline at end of file
+ */
+
+module.exports = async function sleep(sec) {
+  if (typeof sec !== "number") {
+    throw new Error("Sec parameter must be a number", sec);
+  }
+  const defaultWait = 10000;
+  let wait = sec * 1000;
+
+  return new Promise(function (resolve, reject) {
```

Comment:
You don't use the reject. Could you please create a solution, where the reject is also woking?